doc/manpage-newsbeuter.txt

```diff
@@ -41,6 +41,9 @@ OPTIONS
 -i opmlfile, --import-from-opml=opmlfile::
        Import an OPML file
 
+-H homedir, --homedir homedir::
```

Comment:
I also think that `home-dir` might be a better fit given than we're hyphenating things like `config-file`, `url-file` and so on. What do you think?